flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkTest.java

```diff
@@ -78,7 +79,25 @@ public void testRedisClusterDownBehavior() throws Exception {
             .setMaxTotal(1)
             .setMinIdle(1).build();
 
-        testDownBehavior(wrongJedisClusterConfig);
+        RedisSink<Tuple2<String, String>> redisSink = new RedisSink<>(wrongJedisClusterConfig,
```

Comment:
why change this?
its the same code as testDownBehavior no?

Reply:
The exception instance type has changed in the newer version. Previously the cluster and non cluster mode had the same type `JedisConnectionException` and now it is `JedisClusterOperationException`.

https://github.com/redis/jedis/blob/master/docs/3to4.md

```
All JedisCluster constructors now throw a JedisClusterOperationException if unable to connect to any of the provided HostAndPort(s). 
```

Reply:
I think we can change the exception from testDownBehavior to JedisException with our a problem..
or put the 2 exceptions on condition..

Reply:
Good point. I've changed the condition to check for `JedisException`